hack/markdownlint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -eux
+
+IS_CONTAINER=${IS_CONTAINER:-false}
+CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-podman}"
+
+if [ "${IS_CONTAINER}" != "false" ]; then
+  TOP_DIR="${1:-.}"
+  find "${TOP_DIR}" -type d \( -path ./vendor -o -path ./.github \) -prune -o -name '*.md' -exec mdl --style all --warnings {} \+
+else
+  "${CONTAINER_RUNTIME}" run --rm \
+    --env IS_CONTAINER=TRUE \
+    --volume "${PWD}:/capm3:ro,z" \
+    --entrypoint sh \
+    --workdir /capm3 \
```

Comment:
```suggestion
    --volume "${PWD}:/mariadb-image:ro,z" \
    --entrypoint sh \
    --workdir /mariadb-image \
```

Reply:
Ah, sorry, it is supposed to say `/workdir`, not be component specific.